packages/slice-machine/src/features/onboarding/types.ts

```diff
@@ -1,6 +1,7 @@
 import { z } from "zod";
 
 export const onboardingStepStatusesSchema = z.object({
+  createProject: z.literal(true),
```

Comment:
⚠️ With the introduction of another step, can you check what happens when I have completed my onboarding with the previous structure, and then open the application with the new step?

Reply:
yes, I've tested that